jetty-http2/http2-hpack/src/main/java/org/eclipse/jetty/http2/hpack/HpackException.java

```diff
@@ -0,0 +1,54 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2018 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.http2.hpack;
+
+import java.io.IOException;
+
+public abstract class HpackException extends RuntimeException
+{
+    HpackException(String messageFormat, Object... args)
+    {
+        super(String.format(messageFormat, args));
+    }
+    
+    public static class Stream extends HpackException
```

Comment:
I think this class should be renamed `StreamException`. I understand there is duplication (as `HpackException.StreamException`, but it's immediately clear from the name that is an exception and not some data structure you can lookup from `HpackException`.
Also, the JDK always name exceptions ending with `*Exception` apart few historical cases.